Week1/Swarnima_Shishodia/Que10.cpp

```diff
@@ -0,0 +1,18 @@
+#include<iostream>
+using namespace std;
```

Comment:
Write a comment about what the below function is doing. Sometimes interviewer asks to write code on paper. So, it would be a good practice to help him to understand what this function is doing.
For example: # Function calculating GCD